software/externals/cmake/externals.cmake

```diff
@@ -307,6 +307,29 @@ if(NOT APPLE)
 endif()
 
 
+if(USE_SYSTEM_OPENCV)
+  list(REMOVE_ITEM externals
+    opencv
+  )
+  list(REMOVE_ITEM occ-map_depends
+    opencv
+  )
+  list(REMOVE_ITEM libmultisense_depends
+    opencv
+  )
+endif()
```

Comment:
Yeah I think I prefer using the variable (which is empty if not defined) because then you don't have to explicitly undo the dependencies.